### PR TITLE
Add zh_cn translation

### PR DESCRIPTION
**Pull request by FlameFire26**

### CODE REVIEW

File: Common/src/main/resources/assets/chunkbychunk/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+   "block.chunkbychunk.chunkspawner": "区块生成器",
+   "block.chunkbychunk.forestchunkspawner": "森林区块生成器",
+   "block.chunkbychunk.badlandschunkspawner": "恶地区块生成器",
+   "block.chunkbychunk.desertchunkspawner": "沙漠区块生成器",
+   "block.chunkbychunk.junglechunkspawner": "丛林区块生成器",
+   "block.chunkbychunk.mountainchunkspawner": "山峰区块生成器",
+   "block.chunkbychunk.mushroomchunkspawner": "蘑菇岛区块生成器",
+   "block.chunkbychunk.plainschunkspawner": "平原区块生成器",
+   "block.chunkbychunk.rockychunkspawner": "石岸块生成器",
+   "block.chunkbychunk.savannachunkspawner": "热带草原块生成器",
```
Comment (Don-Trueno) [inline Common/src/main/resources/assets/chunkbychunk/lang/zh_cn.json:14]: In line13&14, "块" should be "区块"

File: Common/src/main/resources/assets/chunkbychunk/lang/zh_cn.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+   "block.chunkbychunk.snowchunkspawner": "雪原区块生成器",
+   "block.chunkbychunk.swampchunkspawner": "沼泽区块生成器",
+   "block.chunkbychunk.unstablechunkspawner": "随机区块生成器",
+   "block.chunkbychunk.bedrockchest": "基岩宝箱",
+   "block.chunkbychunk.worldforge": "世界锻造仪仪",
```
Comment (Don-Trueno) [inline Common/src/main/resources/assets/chunkbychunk/lang/zh_cn.json:19]: Repeated "仪" in this line and line27

**Comment by yizexiaomu** [comment]
Extensive modifications have been made on the original basis, and I hope the author can adopt it
[zh_cn.json](https://github.com/immortius/chunkbychunk/files/13858355/zh_cn.json)

**Comment by FlameFire26** [comment]
@yizexiaomu 
- function block better use the same name `"仪"`
- Verbal advancements are debatable to literal translation. (Can ask the author)

**Comment by immortius** [comment]
Just wanted to let you know I'm not ignoring this, I just have received three different translations in quick succession - I'm having a friend who knows the language review them.

**Comment by FlameFire26** [comment]
Never mind, its just my opinion.

**Comment by yizexiaomu** [comment]
> @yizexiaomu
> 
> * function block better use the same name `"仪"`
> * Verbal advancements are debatable to literal translation. (Can ask the author)

I think for Chinese users, it is not necessary to modify its sinicization according to the original text, but to make Chinese users better understand it. I think "function block "should not use `" 仪 ".`

**Comment by immortius** [comment]
Discussing with my friend, we put together https://github.com/immortius/chunkbychunk/blob/1.19.3/Common/src/main/resources/assets/chunkbychunk/lang/zh_cn.json - would be good to get any feedback.

**Comment by immortius** [comment]
Closing as I've added a Chinese language file.

**Pull request closed without merging**